release-doc-in-process/all_vf.TTL

```diff
@@ -666,109 +680,247 @@ vf:inverseLabel a           owl:DatatypeProperty ;
-        vs:term_status      "unstable" ;
-        rdfs:label          "decrement" ;
-        rdfs:comment        "Parent property for all actions that can decrement a resource." .
-
-vf:dropoff  a               vf:Action ;
-        rdfs:label          "dropoff" ;
-        vs:term_status      "unstable"  ;
-        rdfs:comment        "Transported resource or person leaves the process; the same resource or person appeared in the input." .
-
-vf:pickup  a                vf:Action ;
-        rdfs:label          "pickup" ;
-        vs:term_status      "unstable" ;
-        rdfs:comment        "Transported resource or person enters the process; the same resource will appear in the output." .
-
-vf:consume  a               vf:Action ;
+vf:pairsWith a              owl:ObjectProperty ;
+        rdfs:domain         vf:Action ;
+        rdfs:range          vf:Action ;
+        rdfs:comment        "The action that should be included on the other direction of the process, for example accept with modify." ;
+        rdfs:label          "pairs with" ;
+        vs:term_status      "unstable" .
+
+vf:resourceEffect a         owl:ObjectProperty ;
+        rdfs:domain         vf:Action ;
+        rdfs:range          vf:ResourceEffect ;
+        rdfs:comment        "The increment and/or decrement effect the action will have on an inventoried resource." ;
+        rdfs:label          "resource effect" ;
+        vs:term_status      "unstable" .
+
+
+#################################################################
+#    Individuals
+#################################################################
+
+# Actions
+
+vf:accept a                 owl:NamedIndividual ,
+                            vf:Action ;
+        vf:inputOutput      vf:input ;
+        vf:pairsWith        vf:modify ;
+        vf:resourceEffect   vf:noEffect ;
+        rdfs:comment        "In processes like repair or modification or testing, the same resource will appear in the output." ;
+        rdfs:label          "accept" ;
+        vs:term_status      "unstable" .
+
+vf:cite a                   owl:NamedIndividual ,
+                            vf:Action ;
+        vf:inputOutput      vf:input ;
+        vf:pairsWith        vf:notApplicable ;
+        vf:resourceEffect   vf:noEffect ;
+        rdfs:comment        "For example a design file, neither used nor consumed, the file remains available at all times." ;
+        rdfs:label          "cite" ;
+        vs:term_status      "unstable" .
+
+vf:consume a                owl:NamedIndividual ,
+                            vf:Action ;
+        vf:inputOutput      vf:input ;
+        vf:pairsWith        vf:notApplicable ;
+        vf:resourceEffect   vf:decrement ;
+        rdfs:comment        "For example an ingredient or component composed into the output, after the process the ingredient is gone." ;
         rdfs:label          "consume" ;
-        vs:term_status      "testing" ;
-        rdfs:subPropertyOf  vf:decrement ;
```

Comment:
This line should be removed:
---
rdfs:subPropertyOf  vf:decrement ;

Reply:
It's gone, the red and the '-' is a github convention for removed.